.gitignore

```diff
@@ -13,7 +13,7 @@ backend.tf
 [._]sw[a-p]
 session.vimSessionx.vim.netrwhist
 *~
-# tags
+tags
```

Comment:
What is this file?

Reply:
Great question. I'll revert the gitignore, not sure where this crept in from.